src/scripts/scrollHelper.js

```diff
@@ -1,137 +1,137 @@
+export function toStart(container, elem, horizontal, skipWhenVisible) {
+    const pos = getPosition(container, elem, horizontal);
 
-        if (skipWhenVisible && pos.isVisible) {
-            return;
-        }
+    if (skipWhenVisible && pos.isVisible) {
+        return;
+    }
 
-        if (container.scrollTo) {
-            if (horizontal) {
-                container.scrollTo(pos.start, 0);
-            } else {
-                container.scrollTo(0, pos.start);
-            }
+    if (container.scrollTo) {
+        if (horizontal) {
+            container.scrollTo(pos.start, 0);
         } else {
-            if (horizontal) {
-                container.scrollLeft = Math.round(pos.start);
-            } else {
-                container.scrollTop = Math.round(pos.start);
-            }
+            container.scrollTo(0, pos.start);
         }
-    }
-
-    function centerOnFocus(e, scrollSlider, horizontal) {
-        var focused = focusManager.focusableParent(e.target);
-
-        if (focused) {
-            toCenter(scrollSlider, focused, horizontal);
+    } else {
+        if (horizontal) {
+            container.scrollLeft = Math.round(pos.start);
+        } else {
+            container.scrollTop = Math.round(pos.start);
         }
     }
+}
 
-    function centerOnFocusHorizontal(e) {
-        centerOnFocus(e, this, true);
-    }
-    function centerOnFocusVertical(e) {
-        centerOnFocus(e, this, false);
-    }
+function centerOnFocus(e, scrollSlider, horizontal) {
+    const focused = focusManager.focusableParent(e.target);
 
-    return {
-        getPosition: getPosition,
-        centerFocus: {
-            on: function (element, horizontal) {
-                if (horizontal) {
-                    dom.addEventListener(element, 'focus', centerOnFocusHorizontal, {
-                        capture: true,
-                        passive: true
-                    });
-                } else {
-                    dom.addEventListener(element, 'focus', centerOnFocusVertical, {
-                        capture: true,
-                        passive: true
-                    });
-                }
-            },
-            off: function (element, horizontal) {
-                if (horizontal) {
-                    dom.removeEventListener(element, 'focus', centerOnFocusHorizontal, {
-                        capture: true,
-                        passive: true
-                    });
-                } else {
-                    dom.removeEventListener(element, 'focus', centerOnFocusVertical, {
-                        capture: true,
-                        passive: true
-                    });
-                }
+    if (focused) {
+        toCenter(scrollSlider, focused, horizontal);
+    }
+}
+
+function centerOnFocusHorizontal(e) {
+    centerOnFocus(e, this, true);
+}
+function centerOnFocusVertical(e) {
+    centerOnFocus(e, this, false);
+}
+
+export default {
```

Comment:
Shouldn't `centerFocus` be exctracted and exported?

Reply:
and `off`?

Reply:
It is in `centerFocus`.
Should be something like this
```js
export const centerFocus = {
    on() {},
    off() {}
};

export default {
    getPosition,
    centerFocus,
    toCenter,
    toStart
};
```

Reply:
Oh yeah my mistake